sqlite/sqlite.go

```diff
@@ -830,14 +831,11 @@ func (db *DB) NewVoucher(ctx context.Context, ov *fdo.Voucher) error {
 	if err != nil {
 		return fmt.Errorf("error marshaling ownership voucher: %w", err)
 	}
-	table := "mfg_vouchers"
-	if len(ov.Entries) > 0 {
-		table = "owner_vouchers"
-	}
```

Comment:
@ben-krieger the above it seems to be "dead" code as NewVoucher is never called with an ov with 0 entries (it's only called at the end of DI). Just wanted to highlight this in case you had other ideas around this.

Reply:
Given the errors in the unit test, I think we'd need to change the way we approach at vouchers vs onboarded "vouchers"/devices